settings.json

```diff
@@ -1575,6 +1575,46 @@
       "requires": []
     }
   ],
+  "typescript.tsx": [
```

Comment:
As far as I understand, This name does not work.

Reply:
I've been using this convention for years with my own typescript plugin and it worked fine?